### PR TITLE
Clean CoreOS related configs

### PR DESCRIPTION
**Pull request by mihaylovmihail**
# Description

## What

Please be specific and try to describe your thought process. State the obvious, since this might be the first time the reviewer is looking at the code

## Why

Copy (if there is one) the text of the original Trello/JIRA ticket in here, with a link back to it for the curious.

## Anything, in particular, you'd like to highlight to reviewers

Mention here sections of code which you would like reviewers to pay extra attention to .E.g

_Would appreciate a second pair of eyes on the test_  
_I am not quite sure how this bit works_  
_Is there a better library for doing x_  

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [ ] Documentation
- [ ] Breaking change
- [x] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

**Review by dbelev — APPROVED** [review]

File: helm/annotations-publisher/app-configs/annotations-publisher_pac.yaml [hunk 1]
```diff
@@ -1,4 +0,0 @@
-# Values used for the deployed application.
```
Comment (bboykov) [inline helm/annotations-publisher/app-configs/annotations-publisher_pac.yaml:1]: ОК, removing this file, but should there be a helm/annotations-publisher/app-configs/annotations-publisher_pac_dev.yaml file?

**Review by bboykov — APPROVED** [review]

File: helm/annotations-publisher/app-configs/annotations-publisher_pac.yaml (hunk 1 above)
Comment (mihaylovmihail) [reply helm/annotations-publisher/app-configs/annotations-publisher_pac.yaml:1]: There is no DEV Pac cluster. Hope in future there will be no PAC clusters at all.

File: helm/annotations-publisher/values.yaml [hunk 2]
```diff
@@ -9,4 +9,4 @@ image:
   repository: coco/annotations-publisher
   pullPolicy: Always
 
-eksCluster: false
+eksCluster: true
```
Comment (karsov) [inline helm/annotations-publisher/values.yaml:12]: Do we need the "eksCluster" variable at all?

**Review by karsov — COMMENTED** [review]
I think that we can now remove all references to the "eksCluster" variable (values.yaml + app-configs)

**Review by dbelev — APPROVED** [review]

**Review by bboykov — APPROVED** [review]

**Review by karsov — APPROVED** [review]

**Pull request merged**